rest.insomnia.client.appdata.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, this file should be submitted and maintained upstream.

Reply:
Isn't this flatpak-specific? I can bring it up upstream, but I'd prefer to know a bit more about what other uses it has.

Reply:
It's used by any app store on Linux. deb/snap or whatever other packaging format could use it. Even fwupd uses it to describe the firmwares for software centers.

Reply:
Huh, I'd never seen it on any of the distributions I've worked with.

What's the name of this manifest format? I'll ask upstream if they can ship it in their tarball. They already support `snap`, so they may already have a copy somewhere -- but it's not in the git repo, oddly.

Reply:
https://www.freedesktop.org/wiki/Distributions/AppStream/

This is not a blocker, just something that would be nice to be handled at some point.

Reply:
Thanks! I'll include this in the upstream PR and mention it adds value on Debian / Snap too. They publish  packages in those formats, so there's good reason to include it.